thirdparty/civetweb-1.10/src/civetweb.c

```diff
@@ -16237,10 +16237,13 @@ worker_thread_run(struct worker_thread_args *thread_args)
 					mg_free(conn->request_info.client_cert);
 					conn->request_info.client_cert = 0;
 				}
-			}
+			} else {
```

Comment:
Fixed in https://github.com/civetweb/civetweb/pull/630 and released in civetweb 1.11, but that is a rather large rewrite and after all I think just having this patch carries less risk than updating the whole thing.

Reply:
AgreedL https://issues.apache.org/jira/browse/MINIFICPP-966